Extensions/xkit_main.js

```diff
@@ -25,6 +25,22 @@
 			if (XKit.extensions.xkit_main.running === true) {
 				return;
 			}
+
+			if (XKit.page.react === undefined) {
+				XKit.page.react = Boolean($("link[href*='/pop/']").length);
+				$("body").addClass('xkit--react');
+				const waitUntilReactInitialized = () => {
+					if ($('[data-rh]').length === 0) {
+						this.run();
+					} else {
+						setTimeout(waitUntilReactInitialized, 100);
```

Comment:
i'd rather see this as setinterval/clearinterval since the only exit condition is success